package io.github.anvell.keemobile.domain.exceptions

class EntryNotFoundException : Throwable()
